CHANGELOG.md

```diff
@@ -17,7 +20,7 @@
 - Translations for discount errors
 
 ## Changed
-- Spacing between components AB#9013 
+- Spacing between components AB#9013
```

Comment:
I'm not sure what it is exactly... but something doesn't feel right about putting ADO card numbers in a public changelog.

Reply:
@l0ud0gg what do you reckon?